plugins/performance-lab/includes/site-health/cache-control-headers/helper.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Helper functions used for Cache-Control headers for bfcache compatibility site health check.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```

Comment:
Per https://github.com/WordPress/performance/pull/1815:

```suggestion
// @codeCoverageIgnoreStart
if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly.
}
// @codeCoverageIgnoreEnd
```